stories/Avatar/Avatar.story.jsx

```diff
@@ -0,0 +1,86 @@
+
+const TabExample = (
+  <Tab title="Examples">
+    <StoryContainer>
+      <Container fluid>
+        <Title as="h2">Default</Title>
+        <Example component={<Avatar />} code={`<Avatar />`} />
+
+        <Title as="h2">With picture</Title>
+        <Example
+          component={<Avatar picture={encodedPicture} />}
+          code={`<Avatar picture={./picture.png}" />`}
+        />
+
+        <Title as="h2">With notification</Title>
+        <Example
+          component={<Avatar hasNotification />}
+          code={`<Avatar hasNotification />`}
+        />
+
+        <Title as="h2">With Text</Title>
+        <Example
+          component={<Avatar hasNotification text="User Name" />}
+          code={`<Avatar hasNotification text={'User Name'}/>`}
+        />
+
+        <Title as="h2">With href</Title>
+        <Example
+          component={<Avatar text="User Name" href="#user-profile" />}
+          code={`<Avatar href={'#user-profile'} text={'User Name'}/>`}
+        />
+
+        <Title as="h2">Combined with popover</Title>
+        <Example
+          component={
+            <Popover
+              placement="bottom"
+              skin="neutral"
+              trigger={<Avatar hasNotification text="Profile" />}
+            >
+              Some user related info
+            </Popover>
+          }
+          code={`<Popover
+  placement="bottom"
+  skin="neutral"
+  trigger={
+    <Avatar hasNotification text="Profile" />
```

Comment:
**curiosity** if I use a `href` here, what does it happen?

Reply:
I listed the href as a prop now